Makefile

```diff
@@ -97,6 +97,16 @@ container: ## Build the Contour container image
 		$(shell pwd) \
 		--tag $(IMAGE):$(VERSION)
 
+debug-container: ## Build the Contour debug container image
```

Comment:
I assume we'd want to actually push these images to Docker Hub? If so, need to add `make` target and GitHub Actions code for that.